packages/hebao_v1/contracts/modules/security/SecurityModule.sol

```diff
@@ -41,17 +41,14 @@ abstract contract SecurityModule is MetaTxModule
             (_logicalSender == Wallet(wallet).owner() && !isWalletLocked(wallet)),
              "NOT_FROM_WALLET_OR_OWNER_OR_WALLET_LOCKED"
         );
-        SecurityStore ss = controller().securityStore();
-        if (block.timestamp > ss.lastActive(wallet) + MIN_TOUCH_INTERVAL) {
```

Comment:
Brecht, we should try to keep using the same SecurityStore as it is the most important one (WhitelistStore and QuotaStore can be reset for all test users). How does this change impact gas usage, I am thinking to revert this change (line45-47) if possible.

Reply:
@Brechtpd

Reply:
Okay, the only difference is that when the last active actually changes it costs 2,000 gas more. If the timestamp doesn't need to be updated the gas cost remains the same.

Reply:
Let us keep it as is :) A simple and effective change is to use 2 weeks as the MIN_TOUCH_INTERVAL value. What do you think?